
{'date': 'Tue Sep 19 2023 11:58:27.168 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  protao = Elemento(PORTAO_BRONZE), x=10, y=10, w=100, h=100, cena=oceano)
                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 19 2023 11:58:41.23 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    IlhaProibida()
  module <module> line 10
    protao = Elemento(PORTAO_BRONZE, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'PORTAO_BRONZE' is not defined
'''},
{'date': 'Tue Sep 19 2023 11:59:02.702 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    IlhaProibida()
  module <module> line 10
    protao = Elemento(PORTAO_BRONZE, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'PORTAO_BRONZE' is not defined
'''},
{'date': 'Tue Sep 19 2023 12:28:42.810 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 23
  def __init__(self
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 19 2023 12:28:57.39 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 27
    Peao()
  module <module> line 24
    pawn = Elemento(PAWN, x=10, y=10, w=100, h=100, cena=oceano)
NameError: name 'oceano' is not defined
'''},
{'date': 'Tue Sep 26 2023 12:06:49.239 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(loca, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:07:27.206 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''test
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos += (Terreno(local, position, 10))
TypeError: 'Terreno' object is not iterable
'''},
{'date': 'Tue Sep 26 2023 12:08:09.192 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(local, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:08:23.493 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 40
    IlhaProibida()
  module <module> line 23
    self.terrenos.push(Terreno(local, position, 10))
AttributeError: 'list' object has no attribute 'push'
'''},
{'date': 'Tue Sep 26 2023 12:14:03.778 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 36
    IlhaProibida()
  module <module> line 20
    self.terrenos = [Terreno(l, p, i * 100 + 10, oceano) for i, l, p in enumerate(info_terrenos)]
ValueError: need more than 2 values to unpack
'''},
{'date': 'Tue Sep 26 2023 12:28:00.524 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 45
  def mover(x, y)
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 26 2023 12:41:37.879 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Sep 26 2023 12:42:38.476 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Sep 26 2023 12:43:28.879 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    IlhaProibida()
  module <module> line 27
    self.terrenos[2].ocupa(self.pawn)
  module <module> line 39
    peao.mover(self.posx, self.posy)
TypeError: mover() takes 2 positional arguments but more were given
'''},